public/js/jquery.t.js

```diff
@@ -16,6 +16,24 @@
         });
     };
 
+    var setupResponsiveness = function ($queryDiv, $graphDiv, opts)  {
+        var debounce = function (fn, timeout) {
```

Comment:
Doesn't http://underscorejs.org/#debounce serve your purpose? We have underscore, iirc.